sources/src/main/java/com/google/solutions/jitaccess/core/services/RoleDiscoveryService.java

```diff
@@ -291,9 +291,7 @@ public Result<ProjectRole> listEligibleProjectRoles(
     consolidatedRoles.addAll(activatedRoles);
```

Comment:
`consolidatedRoles` is only partially sorted now:

1. It contains JIT-eligible roles first, then MPA-eligible roles, and finally activated roles.
2. Within each of these 3 "buckets", the order is undefined.

(1) is good, but I guess you want each "bucket" to be sorted too. For that, you'd need to change the code above where `mpaEligibleRoles`, `jitEligibleRoles` and `activatedRoles` are populated.

Reply:
Thanks, addressed the comment!